vcpkg/libtorrent/CONTROL

```diff
@@ -0,0 +1,26 @@
+Source: libtorrent
```

Comment:
off topic:
They are finally moving away from the insane format (CONTROL)
https://github.com/microsoft/vcpkg/blob/master/docs/specifications/manifests.md

Reply:
I have no interest in vcpkg yet, so I would limit myself to a minimal touch on it.